packages/desktop-client/src/components/accounts/Header.jsx

```diff
@@ -231,7 +234,7 @@ export function AccountHeader({
                   data-testid="account-name"
                 >
                   {account && account.closed
-                    ? 'Closed: ' + accountName
+                    ? t('Closed: ') + accountName
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should include the `accountName` as a variable here.

I'm not 100% sure about the specific syntax, but the idea is this:

```suggestion
                    ? t('Closed: {accountName}', { accountName })
```

Reply:
This example would require double curly braces (`'Closed: {{ accountName }}'`)